README.md

```diff
@@ -22,8 +26,45 @@ To-be-renamed project repository for the HealthMap/GDO open epidemiology collabo
 
 ![Ingestion functions AWS SAM deploy](https://github.com/globaldothealth/list/workflows/Ingestion%20functions%20AWS%20SAM%20deploy/badge.svg)
 
-## Others
+## Components
+
+- [The data service](data-serving/data-service) in `data-serving/data-service` facilitates CRUD operations with the
+  MongoDB database storing case data.
+- [The curator service](verification/curator-service/api) in `verification/curator-service/api` serves as the backend
+  for the curator portal, which enables curators to view, enter, update, and verify cases; manage data sources and their
+  ingestion; and manage portal access.
+- [The curator UI server](verification/curator-service/ui) in `verification/curator-service/ui` is the frontend for the
+  curator portal.
+
+## Developer documentation
+
+### READMEs
+
+- Getting set up
+    - [Running the stack](dev/README.md)
```

Comment:
Maybe toss a local in there somewhere, or change to `local development`, to distinguish from running in AWS?